sdk/storage/Azure.Storage.Blobs/tests/BlobClientTests.cs

```diff
@@ -1128,7 +1128,7 @@ public async Task UploadAsync_OverwritesDeliberately_Path()
         }
 
         [LiveOnly]
-        [Test]
+       [RecordedTest]
         [Explicit("#10716 - Disabled failing UploadAsync_ProgressReporting live test")]
```

Comment:
There are few tests in this class that are either `LiveOnly` or `Explicit` these shouldn't be ever recorded.